README.md

```diff
@@ -26,14 +26,24 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+React is a UI component-building library for JavaScript. It is representative of a virtual DOM, supporting performance as it will only re-render the difference between the DOM and ‘virtual DOM’, therefore it is data-driven and supports performance when large amounts of data are frequently changed. React enables developers to create ‘components’ (reusable clusters of functional code); these components are functions which return a JSX structure, they can include states, hooks, and relevant functions. It is best practice to provide each component with its own file, this also allows styling components to be produced, keeping all relevant code collectively and stops the potential for pre-declared variable names clashing.
+
 - [ ] What does it mean to _think_ in react?
 
+Thinking in React is the concept of building your application. Considering how your application will look, components and their hierarchy. It is a planning process to develop the relationships of the hierarchy and the components enabling a developer to create meaningful, functional code which is stricter to the concept of DRY. 
+
```

Comment:
Simply put:

To think in React means to think of everything as a piece of a re-usable component.